lte/gateway/c/core/oai/include/service303.h

```diff
@@ -28,10 +28,12 @@
 #define NO_BOUNDARIES 0
 #define NO_LABELS 0
 #define EPC_STATS_TIMER_MSEC 60000  // In milliseconds
+#define EPC_STATS_DISPLAY_TIMER_MSEC 60000 // In milliseconds
```

Comment:
@ardzoht I can plumb this through to a config file if we realize that folks want to tweak the frequency.

Reply:
Sounds good, we can keep this as it is for now

Reply:
Hey @amarpad we need this to value to be configurable. The reason is when we debug issues we need to get faster feedback and know if the UE is there or not, or if the enb is up or down. Otherwise the analysis we do (specially in terravm) becomes impossible. 


Can we set it back to its original value until there is a setting to change that value, please?

Reply:
I think you should add that logging to the S1 thread instead of this thread as S1 is the one that has the latest value. The previous implementation was wrong which was maintaining S1 state in the mme_app thread and could cause crashes.

Reply:
missed to post this here https://github.com/magma/magma/pull/8019
Please comment there if you have any issues on the modification of that value